<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VentanaPNG</class>
 <widget class="QDialog" name="VentanaPNG">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Procesamiento de Imágenes PNG</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(253, 247, 239);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout_botones">
     <item row="0" column="0" colspan="2">
      <widget class="QComboBox" name="combo_filtros">
       <property name="font">
        <font>
         <family>Tw Cen MT Condensed</family>
         <pointsize>14</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_kernel">
       <property name="font">
        <font>
         <family>Tw Cen MT Condensed Extra Bold</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Tamaño Kernel:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLineEdit" name="entrada_kernel">
       <property name="font">
        <font>
         <family>Tw Cen MT Condensed</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QPushButton" name="boton_aplicar">
       <property name="font">
        <font>
         <family>Tw Cen MT Condensed</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(220, 187, 221);</string>
       </property>
       <property name="text">
        <string>Aplicar filtro</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="boton_cargar">
       <property name="font">
        <font>
         <family>Tw Cen MT Condensed</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(220, 187, 221);</string>
       </property>
       <property name="text">
        <string>Cargar Imagen</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="boton_guardar">
       <property name="font">
        <font>
         <family>Tw Cen MT Condensed</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(220, 187, 221);</string>
       </property>
       <property name="text">
        <string>Guardar Resultado</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="boton_volver">
       <property name="font">
        <font>
         <family>Tw Cen MT Condensed</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(220, 187, 221);</string>
       </property>
       <property name="text">
        <string>Volver</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_grafico"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
